tests/jni_api.rs

```diff
@@ -25,6 +28,8 @@ static MATH_ABS_METHOD_NAME: &str = "abs";
 static MATH_TO_INT_METHOD_NAME: &str = "toIntExact";
 static MATH_ABS_SIGNATURE: &str = "(I)I";
 static MATH_TO_INT_SIGNATURE: &str = "(J)I";
+static DEFAULT_EXCEPTION_TYPE: &str = "java/lang/RuntimeException";
+static TEST_EXCEPTION_MESSAGE: &str = "Default exception thrown";
```

Comment:
It is slightly disturbing to see this duplicated constant. :upside_down_face:

Reply:
Duplicated?.. 🤨

Reply:
There are `DEFAULT_EXCEPTION_CLASS` and `DEFAULT_EXCEPTION_TYPE` with the same value.